developers-italia.oas.yaml

```
@@ -814,20 +814,38 @@ paths:
           in: query
           name: all
           description: 'Show all publishers, even the one with "active" set to false'
+        - schema:
+            type: integer
+            format: int32
+            example: 100
+            minimum: 1
+            maximum: 100
+            default: 25
+          in: query
+          name: 'page[size]'
+          description: Limit the amount of results
+        - schema:
+            type: string
+            maxLength: 255
+            pattern: '.*'
+          in: query
+          name: 'page[before]'
+          description: Only results before this cursor
+        - schema:
+            type: string
+            maxLength: 255
+            pattern: '.*'
+          in: query
+          name: 'page[after]'
+          description: Only results after this cursor
     post:
       summary: Create a new Publisher
       description: Create a new Publisher
       tags:
-        - publishers
+        - publisher
```

```suggestion
        - publishers
```

The tag should be `publishers` everywhere.